iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/memory/MemoryManager.java

```diff
@@ -294,35 +299,35 @@ public synchronized void releaseWithOutNotify(IMemoryBlock block) {
    *
    * @param name the name of memory manager
    * @param sizeInBytes the total memory size in bytes of memory manager
-   * @param enable whether memory management is enabled
+   * @param enabled whether memory management is enabled
    * @return the memory manager
    */
   public synchronized MemoryManager getOrCreateMemoryManager(
```

Comment:
The behavior of this method is ambiguous.
If the manager does not exist, it is okay.
However, what if the manager already exists and the method argument conflicts with the manager?
For example, if the existing manager has a size of 10MB, but `sizeInBytes` is 5MB, or the manager is enabled but the parameter `enabled` is false, what will happen?

Reply:
Add it in the method comment.